include/Tunings.h

```diff
@@ -65,9 +65,7 @@ struct Tone
 
     int lineno; // which line of the SCL does this tone appear on?
 
-    Tone() : type(kToneRatio), cents(0), ratio_d(1), ratio_n(1), stringRep("1/1"), floatValue(1.0)
-    {
-    }
+    Tone() : type(kToneRatio), cents(0), ratio_d(1), ratio_n(1), stringRep("1/1"), floatValue(1.0) {}
```

Comment:
Any reason for the format change here Did you clang-format it and thats what clang format did?